src/perl/Permabit/ModuleBase.pm

```diff
@@ -177,6 +179,9 @@ sub buildModule {
   $self->_step(command => "mkdir -p $topdir");
   $self->_step(command => "cp -up $modulePath $topdir");
 
+  if ($self->{useUpstream}) {
+    return
+  }
```

Comment:
Wait, if useUpstream is set, do we even call this function? Do we need this check here?  (If we do call into this, shouldn't be break out before you do any build steps? It should be close t othe top of the function.)

Also, if for some reason we do need this, please add a semicolon after "return".

EDIT: It looks like buildModule() only gets called from loadForSourceRPM(), so the useUpstream flag should prevent us from ever getting here.